.gitignore

```diff
@@ -30,6 +30,7 @@ Makefile.in
 /autom4te.cache
 /bots
 /build/
+/.cache*
```

Comment:
I ran `make` in this branch and I don't get a `.cache` file. Where does it come from?